eureka/lib/readECF.py

```diff
@@ -250,7 +250,11 @@ def copy_ecf(self):
                 else:
                     line_segs = line.strip().split()
                     if line_segs[0]=='inputdir':
-                        new_file.write(line_segs[0]+'\t\t/'+self.inputdir+'\t'+' '.join(line_segs[2:])+'\n')
+                        if self.topdir in self.inputdir:
```

Comment:
This code becomes unnecessary with my previous suggestion.

Reply:
This code could also be unnecessary if inputdir_raw (as described in my previous comment) is correctly defined for all stages - I'm just not sure that it is for all of Stages 1-6

Reply:
Again, are you okay with leaving this as I've written it for now? This currently fixes a bug with Stages 3-6 where the copied ECF's inputdir was previously the topdir+inputdir by accident

Reply:
Yes